ci-operator/jobs/openshift/odo/openshift-odo-main-periodics.yaml

```diff
@@ -0,0 +1,313 @@
+periodics:
+- agent: kubernetes
+  cluster: build01
+  cron: 0 */6 * * *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: main
+    org: openshift
+    repo: odo
+  labels:
+    ci-operator.openshift.io/prowgen-controlled: "true"
+    ci-operator.openshift.io/variant: v4.5
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-odo-main-v4.5-integration-e2e-periodic
+  reporter_config:arout, 10 months ago: • Send periodic job notification to CoreOS slac…
+    slack:
+      channel: '#odo-notifications'
+      job_states_to_report:arout, 9 months ago: • Push notifications types and template
```

Comment:
Why the fields are looking like a statement? Am I missing something here 😕

Reply:
hmm

Reply:
fixed